### PR TITLE
Add apache license

### PR DESCRIPTION
**Pull request by pintea-eugen**
Added a maven plugin that generates Apache License for all java files. The wrong Apache Licenses will also be corrected.
@tethryus @victorursan

### CODE REVIEW

File: cs-amazon/pom.xml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
```
Comment (victorursan) [inline cs-amazon/pom.xml]: how come did you add the parent ? was there a decision made upon this ?

**Comment by victorursan** [comment]
otherwise :+1: good job ;)

File: cs-amazon/pom.xml (hunk 1 above)
Comment (tethryus) [inline cs-amazon/pom.xml]: @pintea-eugen @victorursan The decision was to have the plugin in each module, It was discussed and the changes will be applied.

Comment (victorursan) [inline cs-amazon/pom.xml]: @pintea-eugen  @tethryus I understand we need the plugin, I don't understand when was the parent tag introduced in the discussion... Don't misunderstand me, I agree with the parent tag :)

Comment (pintea-eugen) [inline cs-amazon/pom.xml]: We will use the plugin in each module in order to keep consistency whenever the version changes.

**Comment by tethryus** [comment]
@pintea-eugen Everything looks ok from my side. 👍 
@victorursan WDYT? Can it be merged?

**Comment by victorursan** [comment]
it can :+1:

**Comment by tethryus** [comment]
@pintea-eugen 👍

**Pull request merged**